docs/api/apiv3/paths/storage_files.yml

```diff
@@ -6,12 +6,12 @@ get:
   tags:
     - File links
   description: |-
-    Gets a collection of file data from a storage. 
+    Gets a collection of files from a storage.
     
-    If no `root` context was given, the result is the content of the document root. With `root` context given, the
-    result contains the collections of file data from without the given root file id.
+    If no `root` context is given, the result is the content of the document root. With `root` context given, the
+    result contains the collections of files from within the given root file id.
     
-    If given `root` context is no directory, but a file, the collection contains this file data as a single element. 
+    If given `root` context is no directory, but a file, the collection contains this file as a single element.
```

Comment:
Maybe I still don't get it: If the `root` is a directory, the collection represents its children. If it is a single file then it is a collection of just one item. How does the client know that this time it is not a child? By comparing `root` and collection item? That looks somewhat odd to me.